src/test/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDaoTest.java

```diff
@@ -41,6 +44,8 @@
 
   @Autowired private FireStoreUtils fireStoreUtils;
 
+  @Autowired private ConfigurationService configurationService;
```

Comment:
I thought about that, but I wasn't sure if I could do that with a connected test. I'll look into it some more.

Reply:
After getting it to work with mocking and chatting with @nmalfroy, we decided to go back to the helper method, but move it into testUtils instead of the configurationService. I wanted a solution that would be easily shared between tests and the config service's "insertFault" mechanism made it difficult to mock in some of the other tests.

Reply:
The solution for that may be to use `spy` instead of `mock`. It would be nice to get the fault mechanism out of the config object too but that's a problem for another day.